velox/docs/develop/types.rst

```diff
@@ -73,6 +73,28 @@ Examples:
 * Timestamp(-5000*24*60*60 - 1000, 123456) represents 1956-04-24 07:43:20.000123456
   (5000 days 1000 seconds before epoch plus 123456 nanoseconds).
 
+Floating point types (float, double) have special values negative/positive infinity and
+not-a-number (NaN).
+For negative/positive infinity the following standard floating point semantics apply:
+
+* Positive infinity multiplied by any positive value returns positive infinity.
+* Negative infinity multiplied by any positive value returns negative infinity.
+* Positive infinity multiplied by any negative value returns negative infinity.
+* Negative infinity multiplied by any negative value returns positive infinity.
+* Positive/negative infinity multiplied by 0 returns NaN.
+* Positive/negative infinity is equal to itself.
+* In aggregations, all positive infinity values are grouped together. Similarly, all negative infinity values are grouped together.
```

Comment:
Does this apply to grouping keys? Should we combine this statement with join?

```
Positive infinity and negative infinity are treated as normal values in join and group-by keys.
```

Can we clarify how NaN is handled in join or group-by key?

Reply:
Yes, we can combine this. The aggregation statement about the grouping follows from the equality statement and the fact that they are normal values.